plugins/msr/fu-plugin-msr.c

```diff
@@ -203,7 +203,7 @@ fu_plugin_msr_backend_device_added(FuPlugin *plugin, FuDevice *device, GError **
 						G_LITTLE_ENDIAN,
 						error))
 			return FALSE;
-		if (ver_raw != 0) {
+		if (ver_raw != 0 && ver_raw != G_MAXUINT32) {
```

Comment:
should this maybe be a fail that you return FALSE for this condition?

Reply:
Maybe. I do worry that returning with an error here is going to print something on the console -- it's not really noteworthy. The other thing I worried about is if maybe we wanted to read another offset other than PCI_MSR_IA32_BIOS_SIGN_ID in the future, we'd need to refactor this out and have to swallow the error anyway. I think just skipping the block is probably much simpler.

Reply:
okay